frontend/public/components/container.tsx

```diff
@@ -379,40 +394,60 @@ const ContainerDetails: React.FC<ContainerDetailsProps> = (props) => {
 ContainerDetails.displayName = 'ContainerDetails';
 
 export const ContainersDetailsPage: React.FC<ContainerDetailsPageProps> = (props) => {
+  return (
+    <Firehose
+      resources={[
+        {
+          name: props.match.params.podName,
+          namespace: props.match.params.ns,
+          kind: 'Pod',
+          isList: false,
+          prop: 'obj',
+        } as FirehoseResource,
+      ]}
+    >
+      <ContainersDetailsPage_ {...props} />
+    </Firehose>
+  );
+};
+
+export const ContainersDetailsPage_: React.FC<ContainerDetailsPageProps> = (props) => {
```

Comment:
We are starting to try to avoid trailing underscores in names. It would be better to call this something like `ContainerDetails` to differentiate it from the other `ContainerDetailsPage` component.